requirements.txt

```diff
@@ -1,5 +1,6 @@
 gitpython
 pyyaml
+conda ==4.1.12
 conda-build-all
 conda-build ==2.0.6
-requests !=2.12.0
+requests ==2.11.1 
```

Comment:
The conda pin is necessary due to the api change introduced from conda `4.2.*` ... the problem is in changes to `conda.install.pkgs_dirs` not existing in `4.2.*`, rather its `conda.install.context.pkgs_dirs`